src/helper/response-handler.js

```diff
@@ -58,6 +59,17 @@ function wrapCallback(cb, options) {
         errObj.error_description = errObj.description;
       }
 
+      if (
+        errObj.code === 'blocked_user' &&
```

Comment:
A question - do we need to have this hardcoded check here on 'blocked_user'. 
There might be benefits in making this code generic. If there are other cases in future that need to extend error responses via codes and details, then generic code will require no new code changes in Auth0.js. 
Do you think it makes sense to check if error_codes and error_details are present, then just include them in the response object in generic manner ( and not wrap it in blockedUser object)? 

> {
  code: 'blocked_user',
  description: 'Blocked user.',
  codes: [ 'reason-1', 'reason-2' ],
  details: {
      'reason-1': {
        timestamp: 123786498
      },
      'reason-2': {
        timestamp: 127836389
      }  
  } 
}

or

{
  code: 'blocked_user',
  description: 'Blocked user.',
  errorDetails: {
    codes: [ 'reason-1', 'reason-2' ],
    details: {
      'reason-1': {
        timestamp: 123786498
      },
      'reason-2': {
        timestamp: 127836389
      }
    }
  } 
}

Reply:
@madsharm I like the second option here, primarily because the first one has both `code` and `codes` at the same level on the object, which could be confusing.

Agreed with making it more generic though. Let me make some changes.

Reply:
@madsharm I made those changes in https://github.com/auth0/auth0.js/pull/1084/commits/fcc484c37a6c760a51ce84bea632cc7667cb607b

Let me know what you think

Reply:
LGTM and thanks for doing the change